src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -42,6 +43,8 @@ public enum Setpoint {
   private DoubleLogEntry log_ff_output = new DoubleLogEntry(m_log, "/arm/ff/output");
   private StringLogEntry log_setpoint = new StringLogEntry(m_log, "/arm/setpoint");
 
+  PWM lEDPwm = new PWM(0);
```

Comment:
this port needs to be determined still no?
```suggestion
  private PWM m_led = new PWM(0);
```

Reply:
> this port needs to be determined still no?

maybe, was going off memory, could also be port 1